### ISSUE TITLE
Overall progress

### ISSUE THREAD

**Issue by matthiasdemuzere** [matthiasdemuzere/wrf-lcz-KL#21]
As per @dargueso his suggestion, I closed all other Issues, as it was getting difficult to keep track of what was discussed in there.

From now on we should try to be a bit more structured:
- provide updates to overall progress here
- start a new item for specific bugs / issues.

For now, let's get all simulations started, and then we can discuss about follow-up steps ...

**Comment by dargueso** [comment]
Thanks @matthiasdemuzere 
Both exp B and exp C first jobs submitted, I'll let you know how it goes. Fingers crossed.

**Comment by andreazonato** [comment]
@dargueso  did you managed to run exp C) with BEP+BEM correctly?

**Comment by dargueso** [comment]
@andreazonato Yep, it seems to be running fine at our local machine. Let see how it goes at the supercomputer. At our local machine, I just checked that is running and that it writes out, but it is very slow so I only let it run for a few time steps.

**Comment by dargueso** [comment]
I've uploaded a few sample files for exp B and C in case you want to check them out. Our storage has been down for a week now and I can't access exp A files, but I will upload them as soon as I can.

Please note that outputs are divided into three streams:
wrfout. hourly files, one step per file. Containing mostly 2D variables
wrf3hrly. 3 hourly files. one step per file. Containing mostly 3D variables
wrfprec. 10-min files. 144 steps (one day) per file. Containing precipitation only.

I have not uploaded files for domain 1, only domain 2. You can find them [here](https://drive.google.com/drive/folders/1iaHvzUFsswPYmaW4PzJUXtFlCPs6fIWB?usp=sharing)

**Comment by andreazonato** [comment]
@dargueso , I had a look at the output, and BEP+BEM seems to give reasonable results.
Looking forward to the complete simulation then! =)

**Comment by dargueso** [comment]
Thanks @andreazonato. They are all running :)
Hope the finish by the end of the month, we do have enough CPU time, but the queues get busier as the end of the call period gets closer.
Will keep you updated.
